### PR TITLE
Pm 25 pet page

### PR DESCRIPTION
**Pull request by asching7108**
Implemented pet page. Not yet implemented features:

- picture
- add/remove favorite pet
- news items
Screen Shot 2022-02-11 at 1 25 37 PM

### CODE REVIEW

**Comment by asching7108** [comment]
Btw the page should be responsive, let me know if you see it breaks somehow.

File: src/components/Utils/Utils.js [hunk 1]
```diff
@@ -2,14 +2,17 @@ import { library } from '@fortawesome/fontawesome-svg-core';
...
 import { fab } from '@fortawesome/free-brands-svg-icons';
 import {
-  faCog,
-  faHeart
+  faEnvelope,
+  faGlobe,
+  faHeart,
+  faLocationArrow,
+  faPhone
 } from '@fortawesome/free-solid-svg-icons';
```
Comment (asching7108) [inline src/components/Utils/Utils.js:10]: I use [FontAwesome](https://fontawesome.com/icons) for the icons, which should be free for small projects like ours.

Comment (xHoudek) [reply src/components/Utils/Utils.js:10]: Looks really good! Nice touch

**Comment by xHoudek** [comment]
I like the layout! I think it would make sense if the "About Me" and "Shelter Info" sections were switched. What are your thoughts on that?

File: src/routes/PetPage/PetPage.js [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+  const renderFavoriteIcon = () => {
+    // TODO
+    if (true) {
+      return <span className='text-danger'><FontAwesomeIcon icon='heart' /></span>;
+    }
+    return <span className='text-danger'><FontAwesomeIcon icon={['far', 'heart']} /></span>;
+  }
+
+  const renderPetBio = () => {
```
Comment (xHoudek) [inline src/routes/PetPage/PetPage.js:31]: I think we should maybe have a place to put the animal type: dog, cat, other. Thoughts?

File: src/routes/PetPage/PetPage.js [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+      <div className='col-md m-1'>
+        <h2>
+          Hi, I'm {pet.Name}!
+          <span> </span>
+          {renderFavoriteIcon()}
+          </h2>
+        <table className='table table-sm table-borderless'>
+          <tbody>
+            <tr>
+              <td scope='row'>Breed:</td>
```
Comment (xHoudek) [inline src/routes/PetPage/PetPage.js]: In the case where the animal type is "Other", do you think we should change this from "Breed" to "Species" or something?

A way that might address this comment and the one above is to ditch the label and just have a line under the name that says [Type], [Breed]. So a beagle might say "Dog, Beagle", and a gerbil might say "Other, Gerbil". Thoughts?

File: src/routes/PetPage/PetPage.js [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+              <td>{pet.Breed}</td>
+            </tr>
+            <tr>
+              <td scope='row'>Sex:</td>
+              <td>{pet.Sex}</td>
+            </tr>
+            <tr>
+              <td scope='row'>Age:</td>
+              <td>{pet.Age} yrs</td>
```
Comment (xHoudek) [inline src/routes/PetPage/PetPage.js]: Did we want to handle singular values? like '1 year old' vs '2 years old'

**Review by xHoudek — CHANGES_REQUESTED** [review]
Like I commented before, I really like the look of this page! Just had a few questions and comments. Also, I did check and the page is indeed responsive

File: src/routes/PetPage/PetPage.js (hunk 2 above)
Comment (asching7108) [reply src/routes/PetPage/PetPage.js:31]: Definitely! I totally missed it lol

**Comment by asching7108** [comment]
> I like the layout! I think it would make sense if the "About Me" and "Shelter Info" sections were switched. What are your thoughts on that?

I like it! Makes sense to me as well

File: src/routes/PetPage/PetPage.js (hunk 3 above)
Comment (asching7108) [reply src/routes/PetPage/PetPage.js]: I think that's a great idea! I can make it like a subtitle

File: src/routes/PetPage/PetPage.js (hunk 4 above)
Comment (asching7108) [reply src/routes/PetPage/PetPage.js]: That came across my mind but I ignored it lol I'll handle it haha

**Comment by asching7108** [comment]
Comments addressed!

File: src/routes/PetPage/PetPage.js [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+        <h6>{pet.TypeOfAnimal}, {pet.Breed}</h6>
+        <table className='table table-sm table-borderless'>
+          <tbody>
+            <tr>
+              <td scope='row'>Sex:</td>
+              <td>{pet.Sex}</td>
+            </tr>
+            <tr>
+              <td scope='row'>Age:</td>
+              <td>{pet.Age} {pet.Age === 1 ? 'yr' : 'yrs'}</td>
```
Comment (xHoudek) [inline src/routes/PetPage/PetPage.js]: This is a total nitpick, but I think the shorthand 'yr' looks kind of out of place when everything else on the page is spelled out. Thoughts on using "Year/Years old" instead? Everything else looks great!

Comment (asching7108) [reply src/routes/PetPage/PetPage.js]: Updated!

**Review by xHoudek — DISMISSED** [review]
Looks awesome!

**Comment by xHoudek** [comment]
Ready to merge once the conflict is resolved!

**Comment by asching7108** [comment]
conflicts resolved!

**Comment by asching7108** [comment]
Merged the latest commits in main branch

**Review by xHoudek — APPROVED** [review]

**Pull request merged**